src/track/cueinfo.h

```diff
@@ -58,13 +67,21 @@ class CueInfo {
     void setColor(
             mixxx::RgbColor::optional_t color = std::nullopt);
 
+    CueFlags flags() const {
```

Comment:
getFlags() for consistency within the class? I am aware that we are using both.

For future code I would like to drop the `get` prefix like Qt does. `is` for boolean results is a different use case. This is also the [naming convention recommended for Rust](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)

Reply:
Currently we use an incremental approach to clean up our code (see our diff-based pre-commit hooks). That inevitably leads to inconsistency.

I could also add a follow-up PR that removes the `get` prefixes from the other members. What do you prefer?

Reply:
Let's leave it as is and decide later how to move on. CueInfo might be a good starting point for this cleanup that also needs to be documented.